storage/util.go

```diff
@@ -15,10 +15,12 @@ import (
 	"time"
 )
 
-func (c Client) computeHmac256(message string) string {
+func (c Client) computeHmac256(message string) (string, error) {
 	h := hmac.New(sha256.New, c.accountKey)
-	h.Write([]byte(message))
-	return base64.StdEncoding.EncodeToString(h.Sum(nil))
+	if _, err := h.Write([]byte(message)); err != nil {
```

Comment:
yeah this change causes many changes in the upstream callers whereas the error can be just discarded. wdyt?